src/Controls/src/Core/Compatibility/Handlers/Shell/Android/ShellPageContainer.cs

```diff
@@ -54,6 +54,8 @@ protected override void OnMeasure(int widthMeasureSpec, int heightMeasureSpec)
 				aView.Measure(widthMeasureSpec, heightMeasureSpec);
 				SetMeasuredDimension(aView.MeasuredWidth, aView.MeasuredHeight);
 			}
+			else
+				SetMeasuredDimension(MeasureSpec.GetSize(widthMeasureSpec), MeasureSpec.GetSize(heightMeasureSpec));
```

Comment:
I think here we can just pass in `SetMeasuredDimension(0,0)` 

If the PlatformView isn't present we don't really need/want this to measure out to anything. This is most likely happening during a weird limbo state where the view is being removed or changed out.

Reply:
Done!

Reply:
Perfect! Thank You!